src/tests/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,31 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I believe this file was autogenerated. Like @tonymontaro said yesterday, committing autogenerated files could cause unnecessary merge conflicts. You may want to add this file to .gitignore.

Reply:
Yes, this file was auto-generated. The idea for snapshot testing is to generate a snapshot, save it and then compare it with the react component generated by rendering a component in the test. This would ensure that the UI state does not change unexpectedly by actions from a team member. The convention is to make it human readable and the snapshot artifact should be committed alongside code changes and reviewed as part of our code review process.
I recommend the officaial jest documentation on snapshot testing. [here](https://jestjs.io/docs/en/snapshot-testing)
I will see that the badges are added ASAP.

Thanks.